### PR TITLE
Globalbrightness

### PR DESCRIPTION
**Pull request by dergelm**

### CODE REVIEW

**Comment by rexbutz** [comment]
I had to remove the {{ and }} to get it to show in the drop down:
`<button type='button' class="dropdown-item" onclick='sendMassAction("brightness")'>{{Dimm All Devices}}</button>`
to
`<button type='button' class="dropdown-item" onclick='sendMassAction("brightness")'>Dimm All Devices</button>`

**Comment by versx** [comment]
@dergelm Missing locale entries for `Dimm All Devices`

**Comment by dergelm** [comment]
> @dergelm Missing locale entries for `Dimm All Devices`

whoops 

fixed ;)

**Comment by plinytheelder** [comment]
confirmed that it now works.

File: src/routes/api.js [hunk 1]
```diff
@@ -749,6 +754,14 @@ const get = async (uuid, url) => {
...
             })
             .pipe(fileStream);
+         }else if (isBrightness){
+      request.post(url, (err) => {
+            if (err) {
+                logger('dcm').error(`Error: ${err}`);
+            }
+        }).on('error', (err) => {
+            logger('dcm').error(`Error occurred: ${err}`);
+        });
```
Comment (versx) [inline src/routes/api.js]:
```suggestion
} else if (isBrightness) {
    request.post(url, (err) => {
        if (err) {
            logger('dcm').error(`Error: ${err}`);
        }
    }).on('error', (err) => {
        logger('dcm').error(`Error occurred: ${err}`);
    });
```

**Pull request merged**